CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 
 ## [Unreleased](https://github.com/idealista/prometheus_consul_exporter-role/tree/develop)
-
+### Added
+- Updagrading consul_exporter from [0.3.0](https://github.com/prometheus/consul_exporter/releases/tag/v0.3.0) to [0.7.1](https://github.com/prometheus/consul_exporter/releases/tag/v0.7.1)
```

Comment:
There is a small typo: `Updagraging` -> `Upgrading` :)

Reply:
done